test/explorer/data_frame_test.exs

```diff
@@ -4855,10 +4854,19 @@ defmodule Explorer.DataFrameTest do
       end
     end
 
+    property "can create any dataframe" do
+      check all(
+              dtypes <- Explorer.Generator.dtypes(),
+              rows <- Explorer.Generator.rows(dtypes),
+              max_runs: 1_000
+            ) do
+        assert %DF{} = DF.new(rows, dtypes: dtypes)
+      end
+    end
```

Comment:
I think this is a duplicate of `property "should be able to create a DataFrame from valid rows"` on line 4808.

Reply:
Yeah, just a temporary property when making them was broken as well. I think the final upgrade fixed that tho. Will remove